services/nkp-insights/1.3.4/defaults/cm.yaml

```diff
@@ -413,7 +413,7 @@ data:
       image:
         registry: docker.io
         repository: bitnami/postgresql
-        tag: 15.8.0-debian-12-r14
+        tag: 15.9.0-debian-12-r0
```

Comment:
Should we bump it in the insight repo as well? https://github.com/mesosphere/dkp-insights/

Reply:
checking on it and bump ...Alex..